resources/i18n/qqq.json

```diff
@@ -181,5 +181,6 @@
 	"scroll-next-tab": "Represents the action of scrolling to the next tab of the current tab which toward the end of the tab bar.",
 	"scroll-previous-tab": "Represents the action of scrolling to the previous tab of the current tab which toward the start of the tab bar.",
 	"kiwix-search": "Title text for a list of search results, which notes to the user those are from Kiwix's Search Engine",
-	"multi-zim-search": "Represents the action of searching suggrestion results from one or more ZIM files in the library."
+	"multi-zim-search": "Represents the action of searching suggrestion results from one or more ZIM files in the library.",
+	"select-all": "Represents the action of selecting all ZIM in the library to be search targets for suggestions."
```

Comment:
The too specific description of this message id suggests that it should be renamed to "select-all-zims". Or you can leave the message id as is but remove any reference to ZIMs from its description (so that it can be reused in other contexts too).

Reply:
Done